### PR TITLE
Add bytearray.count

### PR DESCRIPTION
**Pull request by talhof8**

### CODE REVIEW

File: tests/snippets/bytearray.py [hunk 1]
```diff
@@ -71,3 +71,21 @@
 a.append(100)
 assert len(a) == 6
 assert a.pop() == 100
+
+a = bytearray(b'hey there, what is up?')
+assert(a.count(b'h', 3))
+assert(a.count(bytearray(b'h'), 3))
+assert(a.count(104, 3))
+try:
```
Comment (windelbouwman) [inline tests/snippets/bytearray.py]: For this type of contraption, we have this:
```python
from testutils import assertRaises

with assertRaises(TypeError):
    raise TypeError  here
```

**Comment by windelbouwman** [comment]
@talhof8 thank you for contributing to this project!

File: tests/snippets/bytearray.py (hunk 1 above)
Comment (talhof8) [reply tests/snippets/bytearray.py]: Seems about right, resolved :)

**Comment by talhof8** [comment]
https://github.com/RustPython/RustPython/pull/916 added `bytearray.count()`, so this PR is no longer relevant.

**Pull request closed without merging**